tasks/publish-pyxis-repository/publish-pyxis-repository.yaml

```diff
@@ -87,7 +87,10 @@ spec:
 
         # Disable trace logging to avoid leaking of cert+key
         set +x
+        # These variables come from the tekton env so disable shellcheck on them
+        # shellcheck disable=SC2154
```

Comment:
The shellcheck reference says that you could use  `${var:?}` (or some other suggestions) instead of disabling. Could you try them, maybe? 

https://www.shellcheck.net/wiki/SC2154

> Exceptions:
>ShellCheck intentionally does not attempt to figure out runtime or dynamic assignments like with source "$(date +%F).sh" or eval var=value. See [SC2034](https://www.shellcheck.net/wiki/SC2034) for an extended discussion of why this is the case.
>
> If you know for a fact that the variable is set, you can use ${var:?} to fail if the variable is unset (or empty), initialize it to a default value if uninitialized with : "${var:=}", or explicitly initialize/declare it with var="" or declare var. You can also disable the message with a [directive](https://www.shellcheck.net/wiki/directive).

Reply:
It seems switching them to all uppercase makes checkton happy, so I did that

Reply:
heh, who would have thought :)